src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/CoreCLR/WebRequestPSCmdlet.CoreClr.cs

```diff
@@ -57,7 +73,7 @@ public abstract partial class WebRequestPSCmdlet : PSCmdlet
         /// <param name="response">Instance of a WebResponse object to be processed</param>
         internal abstract void ProcessResponse(HttpResponseMessage response);
 
-        #endregion Abstract Methods
+#endregion Abstract Methods
```

Comment:
Generally, you should not mix formatting changes with functional changes.  Have them as separate PRs to make them more readable and reviewable.